CONTRIBUTING.md

```diff
@@ -4,70 +4,68 @@
+ - need help setting up your registry
+ - can't figure out something
+ - are not sure what's going on or what your problem is
 
 Then please do not open an issue here yet - you should first try one of the following support forums:
 
- * irc: #docker-distribution on freenode
- * mailing-list: <distribution@dockerproject.org> or https://groups.google.com/a/dockerproject.org/forum/#!forum/distribution
+ - irc: #docker-distribution on freenode
+ - mailing-list: <distribution@dockerproject.org> or https://groups.google.com/a/dockerproject.org/forum/#!forum/distribution
 
 ## Reporting an issue properly
 
 By following these simple rules you will get better and faster feedback on your issue.
 
- - [ ] search the bugtracker for an already reported issue
+ - search the bugtracker for an already reported issue
 
 ### If you found an issue that describes your problem:
 
- - [ ] please read other user comments first, and confirm this is the same issue: a given error condition might be indicative of different problems - you may also find a workaround in the comments
- - [ ] please refrain from adding "same thing here" or "+1" comments
- - [ ] you don't need to comment on an issue to get notified of updates: just hit the "subscribe" button
- - [ ] comment if you have some new, technical and relevant information to add to the case
+ - please read other user comments first, and confirm this is the same issue: a given error condition might be indicative of different problems - you may also find a workaround in the comments
+ - please refrain from adding "same thing here" or "+1" comments
+ - you don't need to comment on an issue to get notified of updates: just hit the "subscribe" button
+ - comment if you have some new, technical and relevant information to add to the case
 
 ### If you have not found an existing issue that describes your problem:
 
- - [ ] create a new issue, with a succinct title that describes your issue:
-   * bad title: "It doesn't work with my docker"
-   * good title: "Private registry push fail: 400 error with E_INVALID_DIGEST"
- - [ ] copy the output of `docker version`
- - [ ] copy the output of `docker info`
- - [ ] copy your Registry version
- - [ ] copy the command line you used to launch your Registry
- - [ ] restart your docker daemon in debug mode (add `-D` to the daemon launch arguments)
- - [ ] reproduce your problem and get your docker daemon logs showing the error
- - [ ] if relevant, copy your registry logs that show the error
- - [ ] provide any relevant detail about your specific Registry configuration (eg: storage backend used)
- - [ ] indicate if you are using an enterprise proxy, Nginx, or anything else between you and your Registry
+ 1. create a new issue, with a succinct title that describes your issue:
+   - bad title: "It doesn't work with my docker"
+   - good title: "Private registry push fail: 400 error with E_INVALID_DIGEST"
+ 2. copy the output of `docker version`, `docker info`, and your Registry version
```

Comment:
I don't know about this. I like your compacter form, but people might just miss the last two